# Deprecated
Header header
uint8 address
uint8 remote_address
uint32 stamp_us
float32 range
uint16[2] std_noise
uint16[2] preamble_acc_count
uint16[2] first_path_index
uint16[2] first_path_amp_1
uint16[2] first_path_amp_2
uint16[2] first_path_amp_3
uint16[2] channel_impulse_response_power
uint16[2] prf
